cmd/vcluster/cmd/values/v1/base_values.go

```diff
@@ -0,0 +1,349 @@
+package values
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	networkingv1 "k8s.io/api/networking/v1"
+	policyv1 "k8s.io/api/policy/v1"
+)
+
+type BaseHelm struct {
+	GlobalAnnotations    map[string]string      `json:"globalAnnotations,omitempty"`
+	Pro                  bool                   `json:"pro,omitempty"`
+	EnableHA             bool                   `json:"enableHA,omitempty"`
+	Headless             bool                   `json:"headless,omitempty"`
+	DefaultImageRegistry string                 `json:"defaultImageRegistry,omitempty"`
+	Plugin               map[string]interface{} `json:"plugin,omitempty"`
+	Sync                 SyncValues             `json:"sync,omitempty"`
+	FallbackHostDNS      bool                   `json:"fallbackHostDns,omitempty"`
+	MapServices          MapServices            `json:"mapServices,omitempty"`
+	Proxy                ProxyValues            `json:"proxy,omitempty"`
+	Vcluster             VclusterValues         `json:"vcluster,omitempty"`
```

Comment:
Can we rename all the Vcluster stuff into VCluster? Vcluster looks really awkward